samples/tab-sso/nodejs/README.md

```diff
@@ -59,6 +59,7 @@ Your tab needs to run as a registered Azure AD application in order to obtain an
         * Add a platform
         * Select `Single-page application`
         * Create a redirect URI in the format of: `https://contoso.ngrok.io/Home/BrowserRedirect`.
+        * Enable Implicit Grant by selecting `Access Tokens` and `ID Tokens`.
```

Comment:
Putting this back? I think this shouldn't be here at all in the readme as it shouldn't be used by MSAL.js v2

Reply:
Yes correct, this will not be required. Removed this